src/Cassandra/Data/Linq/ClientProjectionCqlQuery.cs

```diff
@@ -54,7 +54,7 @@ internal override IEnumerable<TResult> AdaptResult(string cql, RowSet rs)
             if (!_canCompile)
             {
                 var mapper = MapperFactory.GetMapperWithProjection<TResult>(cql, rs, _projectionExpression);
-                result = rs.Select(mapper);
+                result = Enumerable.Select(rs, mapper);
```

Comment:
I won't comment more on this to avoid spamming the same comment

Reply:
https://github.com/datastax/csharp-driver/pull/617#discussion_r2047004500